packages/entity-detail/src/components/DetailForm/DetailForm.js

```diff
@@ -128,6 +130,7 @@ DetailForm.propTypes = {
   anyTouched: PropTypes.bool,
   formErrors: PropTypes.object,
   valid: PropTypes.bool,
+  isDirty: PropTypes.bool,
```

Comment:
the save button uses "anyTouched" as well. `isDirty` could there be more suitable as well. Remove any unused properties at the end.